docs/src/man/getting_started.md

```diff
@@ -23,6 +23,21 @@ Nullable()
 Nullable(1) + Nullable()
 ```
 
+The get() function can be used to extract the value from the `Nullable` wrapper.
+
+example :
+```julia
+julia> a = Nullable{String}("14:00:00")
+Nullable{String}("14:00:00")
+
+julia> c = get(a)
```

Comment:
ok :) 
I did not understand what you mean by "add a reference to the manual?" do you mean I should insert a link (just paste http://docs.julialang.org/en/stable/manual/types/#nullable-types-representing-missing-values) or do I have to add the example of the get() function in the stable manual ? Sorry, I am very new to github.